### DISCUSSION TITLE
How do you add a custom link to the side navigation?

### DISCUSSION THREAD

**Discussion by joshbuker** [MrGreensWorkshop/MrGreen-JekyllTheme#167] [How to] [answered]
I want to add my photography subdomain as a link with a custom font-awesome icon to the side navigation, but it's not super clear how to do that.

Is this currently supported?

**Comment by MrGreensWorkshop** [comment, marked as answer]
All navigation links (side and top) are generated from the [front matter](https://jekyllrb.com/docs/front-matter/) of the pages.

If you want to add a custom link with no page, please add your link like below to the [_includes/default/nav/navigation-buttons.html](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/blob/main/_includes/default/nav/navigation-buttons.html) file.

`<li><a href="/tabs/about.html" class="hover-effect"><i class="fa-fw fa fa-user-o" aria-hidden="true"></i>About</a></li>`

**Reply by joshbuker** [reply]
This worked! Thanks. :slightly_smiling_face:

**Reply by MrGreensWorkshop** [reply]
I'm glad I could help.
